core/templates/components/question-directives/question-player/question-player.component.html

```diff
@@ -25,9 +25,9 @@ <h4 class="practice-questions-coming-soon-header">Coming Soon!</h4>
     <div class="oppia-question-player-card question-player-results-header">
       <h3 class="oppia-question-player-header protractor-test-practice-complete-message" ng-if="$ctrl.testIsPassed" translate="I18N_QUESTION_PLAYER_TEST_PASSED"></h3>
       <h3 class="oppia-question-player-header" ng-if="!$ctrl.testIsPassed" translate="I18N_QUESTION_PLAYER_TEST_FAILED"></h3>
-      <score-ring score="$ctrl.totalScore"
-                  test-is-passed="$ctrl.testIsPassed">
-      </score-ring>
+      <oppia-score-ring [score]="$ctrl.totalScore"
+                        [is-test-passed]="$ctrl.testIsPassed">
+      </oppia-score-ring>
```

Comment:
Do you feel like `is-test-passed` is the better name? `is-test-passed` sounds more like a function than a property to me.

Reply:
Done

Reply:
Updated the screenshots in description